script/server/config/file.properties

```diff
@@ -0,0 +1,27 @@
+store.model=file
+store.file.dir=sessionStore
+store.file.maxBranchSessionSize=16384
+store.file.maxGlobalSessionSize=512
+store.file.fileWriteBufferCacheSize=16384
+store.file.sessionReloadReadSize=100
+store.file.flushDiskMode=async
+store.db.datasource=druid
+store.db.dbType=mysql
+store.db.driverClassName=com.mysql.jdbc.Driver
+store.db.url=jdbc:mysql://127.0.0.1:3306/seata
+store.db.user=mysql
+store.db.password=mysql
+store.db.minConn=5
+store.db.maxConn=100
+store.db.globalTable=global_table
+store.db.branchTable=branch_table
+store.db.lockTable=lock_table
+store.db.queryLimit=100
+store.db.maxWait=5000
+store.redis.host=127.0.0.1
+store.redis.port=6379
+store.redis.password=
```

Comment:
= ""

Reply:
`store.redis.password= ""`  This config value is  """"